Werkfile

```diff
@@ -0,0 +1,29 @@
+let mdbook = which "mdbook"
+let werk-vscode-version = shell "jq --raw-output .version werk-vscode/package.json" 
+
+build "/werk-vscode/werk-{werk-vscode-version}.vsix" {
+    from glob "werk-vscode/**/*"
+    run "npm --prefix werk-vscode run package -- --out ../target/werk-vscode"
+}
+build "/book/book" {
+    run "{mdbook} build book --dest-dir book/book"
+}
+
+task vscode-install-extension {
+    let vsix = "/werk-vscode/werk-{werk-vscode-version}.vsix"
+    build vsix
+    run "code --install-extension <vsix>"
+}
+
+task cli-install {
+    run "cargo install --path werk-cli"
```

Comment:
Funnily, this will usually clobber the process doing it, which will always fail on Windows. Might be nice for completeness, still.